package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sqlite",
-  "version": "5.1.1",
```

Comment:
Let's make it a major version bump to 6.0.0 since we're going to remove the old `migrations.ts` file

Reply:
Okay. Why you decided to change your mind?)

Reply:
we're removing the old migrate.ts file which someone may use directly and replacing it with this class